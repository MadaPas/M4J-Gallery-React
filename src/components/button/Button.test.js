/* eslint-disable no-console */
import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Button from './Button';

test('Luxon', () => {
  const rel = 'prev';
  const url = 'link.com';
  const tree = renderer
    .create(<Button key={rel} url={url} rel={rel} fetchData={data => console.log(data)} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
