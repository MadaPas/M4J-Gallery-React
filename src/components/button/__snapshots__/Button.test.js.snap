// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Luxon 1`] = `
<button
  className="prev-button button"
  onClick={[Function]}
  type="button"
  value="link.com"
>
  &lt;
</button>
`;
